From 135f7bf1043ca2a48d88d920fd46f3fe0097f359 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Sat, 10 Jan 2015 11:18:50 +1300
Subject: [PATCH] wgl: Fix an extra break statement.

This would have leaked our TLS indices on process exit.

Fixes #32
---
 src/dispatch_wgl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/dispatch_wgl.c b/src/dispatch_wgl.c
index 5b3a8ef..2bc5fc1 100644
--- a/src/dispatch_wgl.c
+++ b/src/dispatch_wgl.c
@@ -130,7 +130,6 @@ DllMain(HINSTANCE dll, DWORD reason, LPVOID reserved)
 
         data = TlsGetValue(wgl_tls_index);
         LocalFree(data);
-        break;
 
         if (reason == DLL_PROCESS_DETACH) {
             TlsFree(gl_tls_index);
